Local environment crash

Running with the `README.md` commands as well as in a container fail with the same error:

``` bash
module.js:340
    throw err;
    ^
Error: Cannot find module '../local-settings.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/scott/Dropbox/Projects/cvr-view/routes/index.js:12:37)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/scott/Dropbox/Projects/cvr-view/app.js:15:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
error: Forever detected script exited with code: 8
```

Seems pretty cut and dry. Did you forget to commit a file or add instructions? I'm assuming the local-settings.json is for Github access keys and such.

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ The required configuration information is for GitHub OAuth and the MongoDB conne
 
 ## Running with forever
 
-`npm install`
-`npm install forever -g`
-`forever start bin/www`
+- `npm install`
+- `npm install forever -g`
+- `forever start bin/www`
 

